src/GetPos.java

```diff
@@ -0,0 +1,54 @@
+import java.util.Scanner;
+
+public class GetPos {
+	String HelpMenu;
+	int[] Positions;
+	private static Scanner in;
+	public GetPos(int[] Positions){
+		SetPosition(Positions);
+	}
+	public void SetPosition(int[] Pos) {
+        in = new Scanner(System.in);
+        
+        //Setting up local variables 
+        //stringCheck1 and stringCheck2 are to determine the input numbers
+        //userPosition1 and userPosition2 are player's starting x and y positions
+        //movement1 and movement2 are player's movements in the x and y coordinates
+
+        	String userPosition1;
+        	String userPosition2;
+
+        	
+        	//User entering starting X and Y positions
+        	System.out.println("Enter Starting X Position:");
+        	userPosition1 = in.nextLine();
+        	System.out.println("Enter Starting Y Position:");
+        	System.out.println();
+        	userPosition2 = in.nextLine();
+        	
+        	//Converting starting positions from string to integers
+        	int newUserPositionX1 = Integer.parseInt(userPosition1);
+        	int newUserPositionY1 = Integer.parseInt(userPosition2);

```

Comment:
Take a look at the getInstruction() method in the TextInputReciever class. There are some catches you'll want to implement. (i.e. make sure the input is able to be parsed to integer)